src/main/java/net/sf/jabref/logic/exporter/ExportFormat.java

```diff
@@ -179,23 +179,12 @@ private Reader getReader(String filename) throws IOException {
         return reader;
     }
 
-    /**
-     * Perform the export of {@code database}.
-     *
-     * @param databaseContext the database to export from.
-     * @param file       the file to write the resulting export to
-     * @param encoding   The encoding of the database
-     * @param entries    Contains all entries that should be exported.
-     * @throws IOException if a problem occurred while trying to write to {@code writer}
-     *                     or read from required resources.
-     * @throws Exception   if any other error occurred during export.
-     * @see net.sf.jabref.logic.exporter.IExportFormat#performExport(BibDatabaseContext, String, Charset, List)
-     */
+
     @Override
     public void performExport(final BibDatabaseContext databaseContext, final String file,
             final Charset encoding, List<BibEntry> entries) throws Exception {
-        Objects.requireNonNull(databaseContext);
-        Objects.requireNonNull(entries);
+        Objects.requireNonNull(databaseContext, "Database Context must not be null!");
```

Comment:
To be honest, in my opinion these string messages don't add more relevant information.